### ISSUE TITLE
Provide the trained models for various scenarios.

### ISSUE THREAD

**Issue by amjjbonvin** [DeepRank/deeprank2#602]
**Is your feature request related to a problem? Please describe.**

In the previous DeepRank repo we were providing the pre-trained models. 
These are missing here. It would improve the usability of the code.

**Comment by gcroci2** [comment]
> **Is your feature request related to a problem? Please describe.**
> 
> In the previous DeepRank repo we were providing the pre-trained models. These are missing here. It would improve the usability of the code.

Do you mean [this](https://github.com/DeepRank/deeprank/tree/master/paper_pretrained_models) folder? If yes, for providing the same pre-trained models we should re-train them, in order to save them again in a format which is compatible with DeepRank2 API.

**Comment by amjjbonvin** [comment]
Yes - and also the models of DeepRank-GNN. Those models are needed is users want to use it for scoring their own models as we describe in the original papers.

**Comment by gcroci2** [comment]
We are reviewing and cleaning the open issues on deeprank2, this one is a very nice to have, but we don't have the resources to pick it up at the moment. Just a heads up in case you want/can to assign this to someone in the lab @LilySnow
